src/components/GroupMenuComponent.js

```diff
@@ -153,11 +158,12 @@ function GroupMenuComponent({
     groupsIndex,
     groupsData,
     'selections',
+    direction,
     onProcessItem,
     'nothingToSelect'
   );
 
-  const activeEntry = generateMenuItem(contextId, onProcessItem);
+  const activeEntry = generateMenuItem(contextId, direction, onProcessItem);
```

Comment:
This qualifies as a breaking change, should we reflect that in the version change? and did you verify that it doesn't break any part of the codebase where it is used?

Reply:
Yes, I've made fix to wA also.  Will bump version.

Reply:
A breaking change requires a major version change and you did a minor version change